src/backups2datalad/__main__.py

```diff
@@ -176,7 +176,7 @@ async def wrapped(
 @click.option("-w", "--workers", type=int, help="Number of workers to run concurrently")
 @click.option(
     "--zarr-mode",
-    type=click.Choice(list(ZarrMode)),
+    type=click.Choice(list(ZarrMode), case_sensitive=False),
```

Comment:
note that we do get it working with recent one but as `asset_checksum` not `asset-checksum` as before... I will make it more explicit -- will RF to use `enum.auto` and get away from `__str__`...